### PR TITLE
Huaweiads: changed deprecated method "imptrackers" to "eventtrackers"

### PR DESCRIPTION
**Pull request by clho40**

### CODE REVIEW

File: adapters/huaweiads/huaweiads.go [hunk 1]
```diff
@@ -1187,10 +1188,17 @@ func (a *adapter) extractAdmNative(adType int32, content *content, bidType openr
 				linkObject.ClickTrackers = append(linkObject.ClickTrackers, monitor.Url...)
 			}
 			if monitor.EventType == "imp" {
-				nativeResult.ImpTrackers = append(nativeResult.ImpTrackers, monitor.Url...)
+				for _, url := range monitor.Url {
+					var eventTracker nativeResponse.EventTracker
+					eventTracker.Event = native1.EventTypeImpression
+					eventTracker.Method = native1.EventTrackingMethodImage
+					eventTracker.URL = url
```
Comment (onkarvhanumante) [inline adapters/huaweiads/huaweiads.go:1195]: iterator var `url` refers  to same address for value in each iterations. so prefer using index variable to assign value for `eventTracker.URL` 

```
for idx, url := range monitor.Url {
      ...
     eventTracker.URL = monitor.Url [idx]
     ...
}

```

Comment (clho40) [reply adapters/huaweiads/huaweiads.go]: Done. But why tho? Is this some kind of best practices when it comes to golang?

Comment (clho40) [reply adapters/huaweiads/huaweiads.go]: May I know what is the benefit of such implementation?

**Comment by clho40** [comment]
not sure why the first test failed

**Review by Sonali-More-Xandr — APPROVED** [review]

File: adapters/huaweiads/huaweiads.go (hunk 1 above)
Comment (onkarvhanumante) [reply adapters/huaweiads/huaweiads.go]: loop variable `url`  gets overridden in every iteration with new value at same memory location. so it is advisible to use  index variable for assignment

**Pull request merged**